### ISSUE TITLE
Missing ScanTailor-CLI or at least (some) command line parameters to start main window

### ISSUE THREAD

**Issue by vboehm** [4lex4/scantailor-advanced#171]
I'm not using ScanTailor to scan books with hundreds of pages but to scan my correspondence. Many documents which mostly have 1 to 10 pages. All pages of one document are scanned into a single directory and so I wrote a short wrapper program, which is called with a list of all these directories, and itself uses _eog_ (eye of gnome), _scantailor-cli_, in special cases _scantailor_, _tiff2pdf_, and _ocrmypdf_ to convert these scans to nice annotated PDFs.

Now _scantailor-cli_ is not available anymore and I tried to replace it with _scantailor_ advanced. But since there is no description if there are any command line parameters (no man page, no '-h', '-?' or '--help' for any information), I have to

- start scantailor
- choose new project
- select a directory
- select all scanned images
- click about 12 times at the different stages of the _ScanTailor_ process
- before the last step I always have to change the 600 dpi output resolution to 300 dpi
- store everything
- and leave the program

Since the former _scantailor-cli_ call ran in the background, the new process takes at least 10 times longer than the old one (feels like 100 times).
By trial I found out, that I can call _scantailor_ with the name of a configuration file (the one you get when you select 'Save Project') and I managed to modify my program to create a rudimentary version of this configuration file (without the complex _filters_ node) to start the program skipping the boring questions about directories and files.

Could you provide a (preferably documented) way to start _ScanTailor_ with all information it needs for its work? Either by command line or a faked configuration file.

Or - the best solution - (please, please, please) provide the missing _scantailor-cli_ again?

Thank you
Volker

**Comment by mara004** [comment]
Good point, but it seems that ScanTailor Advanced is unmaintained, unfortunately. The CLI has been requested repeatedly in the past, e. g. in #1 or #118. The only point that you can fix easily is output DPI, which is configurable under `Tools -> Default parameters ... -> Custom -> Output -> Output Resolution`
Maybe you'd be better off with a different branch of ScanTailor, like [Universal](https://github.com/trufanov-nok/scantailor-universal) or [Experimental](https://github.com/Tulon/scantailor/tree/experimental), which do have a working CLI. Or, perhaps even better, change your scripts to use [unpaper](https://github.com/unpaper/unpaper) which is well-documented, more powerful and gets better support. Another option would be to give [gscan2pdf](http://gscan2pdf.sourceforge.net/) a try, which can do lots of automatic scan improvements (via unpaper) and even OCR.
I hope this helps with your correspondence.

**Comment by mara004** [comment]
> Now _scantailor-cli_ is not available anymore and I tried to replace it with _scantailor_ advanced.

@vboehm It is still available, though not in this version of ScanTailor. I guess `scantailor-universal-cli` would be best for you; it should work in exactly the same way as the `scantailor-cli` you're used to from the original program by @tulon.
Simply go to the ScanTailor Universal releases page and follow the installation instructions, or build it from source (depending on what distribution you are using).
